recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+      "IFUNC523            56485.155266125620983              0.0116919664\n",
+      "IFUNC524            56488.655266096655396              0.0112607949\n",
+      "IFUNC525            56492.155266067682533              0.0108416747\n",
+      "IFUNC526            56495.655266029607446              0.0104323877\n",
+      "IFUNC527            56499.155265991525084              0.0100306212\n",
+      "IFUNC528            56502.655265966553998              0.0096341929\n",
+      "IFUNC529            56506.155265941575635              0.0092410735\n",
+      "IFUNC530             56509.65526595153642               0.008849292\n",
+      "IFUNC531            56513.155265961504483              0.0084568539\n",
+      "IFUNC532             56516.65526598867291              0.0080617905\n",
+      "IFUNC533             56520.15526601584861              0.0076623975\n",
+      "IFUNC534            56523.655266085879703              0.0072576367\n",
+      "IFUNC535            56527.155266155910795              0.0068475859\n",
+      "IFUNC536            56530.655265990564658              0.0064337758\n",
+      "IFUNC537             56534.15526582521852              0.0060192501\n",
+      "IFUNC538             56537.65526601360034              0.0056082516\n",
+      "IFUNC539            56541.155266201974882              0.0052055435\n",
+      "IFUNC540             56544.65526614661212              0.0048154973\n",
+      "IFUNC541             56548.15526609124936              0.0044411843\n",
+      "IFUNC542            56551.655265946988948              0.0040837503\n",
+      "IFUNC543            56555.155265802728536              0.0037423212\n",
+      "IFUNC544             56558.65526598260476              0.0034145784\n",
+      "IFUNC545             56562.15526616247371              0.0030979705\n",
+      "IFUNC546            56565.655266010966443              0.0027913441\n",
+      "IFUNC547              56569.1552658594519              0.0024966239\n",
+      "IFUNC548             56572.65526600497833              0.0022200923\n",
+      "IFUNC549            56576.155266150504758              0.0019728553\n",
+      "IFUNC550            56579.655266085210314              0.0017702185\n",
+      "IFUNC551            56583.155266019908595              0.0016299324\n",
+      "IFUNC552            56586.655266019908595              0.0015695325\n",
+      "IFUNC553            56590.155266019908595              0.0016032517\n",
+      "IFUNC554            56593.655266019908595              0.0017391451\n",
+      "IFUNC555            56597.155266019908595              0.0019771089\n",
+      "IFUNC556            56600.655266019908595               0.002308367\n",
+      "IFUNC557            56604.155266019908595              0.0027167581\n",
+      "IFUNC558            56607.655266019908595              0.0031818254\n",
+      "IFUNC559            56611.155266019908595              0.0036833228\n",
```

Comment:
So the output of the ephemeris file is useful. However, all these IFUNC* stuff comes from the fitting of Fermi data with PINT and are not useful to us. 
The thing that I don't know is if I click on the output cell, to reduce it like this: 
![Capture d’écran 2023-03-02 à 16 59 00](https://user-images.githubusercontent.com/108189156/222482208-07d908a5-93f6-4fe9-a6e9-3603bb735ec6.png)
does the the people reading the notebook are going to be able to scroll through the output ? Because if it's the case, we can put it like this.